vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -66,12 +66,19 @@
       namespace: "{{ IRONIC_NAMESPACE }}"
     when: EPHEMERAL_CLUSTER == "minikube"
 
-  - name: Label BMO CRDs.
+  - name: Label baremetalhost CRD to pivot.
     shell: "kubectl label --overwrite crds baremetalhosts.metal3.io {{ item }}"
     with_items:
        - clusterctl.cluster.x-k8s.io=""
     when: CAPM3_VERSION != "v1alpha4"
 
+  - name: Label hostdata CRD to pivot.
+    shell: "kubectl label --overwrite crds hostdata.metal3.io {{ item }}"
+    with_items:
+       - clusterctl.cluster.x-k8s.io=""
+       - clusterctl.cluster.x-k8s.io/move=""
```

Comment:
Should this label also be added to the BMH CRD above? I'm not sure why it is here but not there.

Reply:
@lentzi90  I'm not sure why we don't need move for BMH. But only clusterctl label on HD CRD wasn't enough to pivot it during my tests. And from the description, move label is actually what we need to ensure that CRD that's not controlled/installed via clusterctl, can be pivoted. Ref: https://github.com/kubernetes-sigs/cluster-api/blob/a6edb8e4355897830a1ea6006195db3a3b4f3565/cmd/clusterctl/api/v1alpha3/labels.go#L35

Reply:
I'm guessing that the BMH is linked to a cluster through owner references already so that's probably why we don't need it. Thanks for checking!

Reply:
You are right, BMH is owned by Metal3Machine. And if the object is linked to cluster indirectly (via ownerRef chain) it can still be pivoted. But as said I will confirm soon after my tests